fenix/app/src/main/res/xml/preferences.xml

```diff
@@ -166,7 +166,6 @@
             android:title="@string/preferences_external_download_manager" />
 
         <androidx.preference.SwitchPreference
-            android:defaultValue="true"
```

Comment:
It's hard to sync this default value with what we have in our preferences so it's best of we don't use the default here and let it read it from the pref instead.